enterprise-suite/templates/es-console.yaml

```diff
@@ -72,22 +79,100 @@ data:
     server {
       listen 8080;
       server_name localhost;
+      set $base /usr/share/nginx/html;
 
-      location / {
-        root  /usr/share/nginx/html;
-        index index.html index.htm;
-        try_files $uri$args $uri$args/ /index.html;
+      # use external resolver to lookup backends, cache for 30 seconds
+
+      resolver 127.0.0.1:5353 ipv6=off valid=30s;
+
+      set $prometheus "prometheus-server.{{.Release.Namespace}}.svc.cluster.local.";
+      set $monitorapi "es-monitor-api.{{.Release.Namespace}}.svc.cluster.local.";
+      set $grafana "grafana-server.{{.Release.Namespace}}.svc.cluster.local.";
```

Comment:
This isn't going to work outside of minikube. The fqdn here will change based on the cluster setup. We need to rely on the search domains that k8s installs.

Reply:
omg.

Oh!  At this means the "resolver" needs to do search, so need to test nginx as I don't think it does this in external resolver mode. gah

Reply:
fixed, but .. bummer